modules/core/src/v2/coins/eth.ts

```diff
@@ -722,6 +728,44 @@ export class Eth extends BaseCoin {
       .asCallback(callback);
   }
 
+  /**
+   * Check whether the gas price and gas limit passed in by user are within our max and min bounds
+   * If they are not set, set them to the defaults
+   * @param userGasPrice user defined gas price
+   * @param userGasLimit user defined gas limit 
+   */
+  checkGasPriceAndLimit(userGasPrice?: number, userGasLimit?: number): GasInfo {
+    let gasLimit;
+    let gasPrice;
+
+    if(userGasLimit) {
+      // check whether the number makes sense
+      const gasLimitMax = config.ethGasConfigs.maximumGasLimit;
+      const gasLimitMin = config.ethGasConfigs.minimumGasLimit; 
```

Comment:
Can you lift these out of the `if` statements since they are used in both of them?

Reply:
Not sure what you meant by used in both of them? These two are the gas_limit_ max and min, whereas the two below are gas_price_ max and min. Is this what you are referring to?

Reply:
Ah my mistake, you are right.